internal/jsnum/pseudobigint.go

```diff
@@ -34,20 +39,15 @@ func (value PseudoBigInt) Sign() int {
 
 func ParseValidBigInt(text string) PseudoBigInt {
 	text, negative := strings.CutPrefix(text, "-")
-	return PseudoBigInt{
-		Negative:    negative,
-		Base10Value: ParsePseudoBigInt(text),
-	}
+	return NewPseudoBigInt(ParsePseudoBigInt(text), negative)
 }
 
 func ParsePseudoBigInt(stringValue string) string {
 	stringValue = strings.TrimSuffix(stringValue, "n")
-
```

Comment:
What do you have against my newlines? 😄

Reply:
Just not a fan of blank lines in code blocks. Would rather have comments separating the spans, or a separate function if it's important enough.

Reply:
I can't say I agree but I guess it's fine...